types/progress/index.d.ts

```diff
@@ -138,4 +138,4 @@ declare class ProgressBar {
 }
 declare namespace ProgressBar { }
 
-export = ProgressBar;
+export default ProgressBar;
```

Comment:
Is this change correct? I have a `ts` file with the following that used to work that is now broken (and reverting `@types/progress` is the fix)

```typescript
import * as ProgressBar from "progress";
new ProgressBar(":bar");
```

`progress` doesn't have a `default` export, so this is incorrect

Reply:
Fix is up at #31398